mhctools/base_commandline_predictor.py

```diff
@@ -303,7 +310,11 @@ def predict_peptides(self, peptides):
                     allele=allele,
                     peptide_mode=True,
                     temp_dirname=temp_dirname)
-        return self._run_commands_and_collect_predictions(
+        results = self._run_commands_and_collect_predictions(
             commands=commands,
             input_filenames=input_filenames,
             temp_dir_list=dirs)
+        self._check_result_count(
```

Comment:
suggestion if it's easy to implement (feel free to ignore): would it be possible for the error message to show which (peptide, allele) pairs were missing from the result, instead of just the number? That would make it obvious when e.g. an X in the peptide is causing the issue